too-many-hooks/src/useTally.ts

```diff
@@ -68,41 +68,41 @@ export type UseCountReturn = [
 ]
 
 /**
- * `useCount` hook type
+ * `useTally` hook type
  *
  * @export
- * @typedef {UseCount}
- * @param {UseCountArgs}
- * @returns {UseCountReturn}
+ * @typedef {UseTally}
+ * @param {UseTallyArgs}
+ * @returns {UseTallyReturn}
  */
-export type UseCount = (args: UseCountArgs) => UseCountReturn
+export type UseTally = (args: UseTallyArgs) => UseTallyReturn
 
 /**
  * Returns a numeric `count` and functions to set, increment, decrement, or reset it
  *
  * @example
  * To count up by 2s
  * ```ts
- * const [count, { increment }] = useCount({step: 2})
+ * const [count, { increment }] = useTally({step: 2})
  * ```
  * @example
  * To count down by 1s
  * ```ts
- * const [count, { decrement }] = useCount()
+ * const [count, { decrement }] = useTally()
  * ```
- * @implements {UseCount}
- * @param {UseCountArgs} { initial = 0, step = 1 }
- * @returns {UseCountReturn}
+ * @implements {UseTally}
+ * @param {UseTallyArgs} { initial = 0, step = 1 }
+ * @returns {UseTallyReturn}
  */
-export const useCount: UseCount = ({ initial = 0, step = 1 }: UseCountArgs): UseCountReturn => {
+export const useTally: UseTally = ({ initial = 0, step = 1 }: UseTallyArgs): UseTallyReturn => {
   const [count, set] = useState<number>(initial)
```

Comment:
Let's rename this state and any derivative variable names after "tally" as well

```suggestion
  const [tally, set] = useState<number>(initial)
```